src/core/public/apm_system.ts

```diff
@@ -88,22 +92,15 @@ export class ApmSystem {
     }
   }
 
-  /* Returns the number of resources that were loaded from network (vs. from disk cache) */
-  private getLoadedResourcesCount() {
-    try {
-      return window.performance
-        .getEntriesByType('resource')
-        .filter((r: Record<string, any>) => r.initiatorType === 'script' && r.transferSize > 0)
-        .length;
-    } catch (e) {
-      return -1;
-    }
-  }
-
   /* Close and clear the page load transaction */
   private closePageLoadTransaction() {
     if (this.pageLoadTransaction) {
-      this.pageLoadTransaction.addLabels({ loadedResources: this.getLoadedResourcesCount() });
+      const loadCounts = this.resourceCounter.getCounts();
+      this.pageLoadTransaction.addLabels({
+        loadedResources: loadCounts.networkOrDisk,
```

Comment:
// Keeping naming consistent with other fields in APM server intake. 
nit: "loaded-resources", "cached-resources"